sklearn/svm/classes.py

```diff
@@ -498,7 +498,8 @@ class SVC(BaseSVC):
         Whether to return a one-vs-rest ('ovr') decision function of shape
         (n_samples, n_classes) as all other classifiers, or the original
         one-vs-one ('ovo') decision function of libsvm which has shape
-        (n_samples, n_classes * (n_classes - 1) / 2).
+        (n_samples, n_classes * (n_classes - 1) / 2). However it doesn't influence
+        the multi-class strategy used. This is used to utilize sklearn API.
```

Comment:
I don't think the last sentence is helpful. Would be better to explicitly state that the multi-class strategy used is OVO.

Reply:
Okay. I will change it.